.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
+
+## Related code and/or documentation pull requests
+
+* none
+
+## Documentation
+
+- [ ] User facing documentation not needed
+- [ ] Release notes entry not needed
+
+### Release notes section
+
+- [ ] General
+- [ ] Security
+- [ ] Connector(s)
+- [ ] SPI
+
+Specify which connectors are affected:
```

Comment:
That should go into the PR title.

Reply:
No .. author should be explicit .. e.g. if something affects base jdbc class and some but not all jdbc based connectors.. the contributor and maintainer should assemble that list and add it here, so it is ready for the release notes assembly..

Reply:
It is also not realistic if more than two connectors are affected..

Reply:
so the short version of this should go into title (one connector - easy, many JDBC connectors -- easy, just say "jdbc connectors")
the longer and more precise version would go at the bottom, where rn entry is suggested. awesome, so this point is redundant

remove